### ISSUE TITLE
onnx predictions are not good 

### ISSUE THREAD

**Issue by Sandeep418** [feiyuhuahuo/Yolact_minimal#24]
I converted  res50.pth  (best_29.3_res50_coco_400001.pth) to onnx using export2onnx.py, The predictions with onnx are not good.
![000000020247](https://user-images.githubusercontent.com/40691260/114142688-e1c20180-9930-11eb-82b3-40004ebc9dd8.jpg)
![000000046804](https://user-images.githubusercontent.com/40691260/114142699-e4bcf200-9930-11eb-99e2-56c40176aee9.jpg)
![000000117525](https://user-images.githubusercontent.com/40691260/114142764-f7372b80-9930-11eb-84e6-8a09a3264aba.jpg)

**Comment by feiyuhuahuo** [comment]
![000000087742](https://user-images.githubusercontent.com/32631344/114145776-5ef97080-9949-11eb-83d9-dd432ac6e20b.jpg)

It works fine with me. Have you made any modification? And which is your onnxruntime-gpu version?

**Comment by Sandeep418** [comment]
I didn't modify anything. I am using onnxruntime-gpu version is 1.6.0
![Screenshot from 2021-04-09 19-00-08](https://user-images.githubusercontent.com/40691260/114193177-e2c45480-996b-11eb-8439-58c64993dc0d.png)

**Comment by feiyuhuahuo** [comment]
The config is not right. If you want to export res_50 weight, you should:
`python export2onnx.py --weight=best_29.3_res50_coco_400001.pth`, then a res50_coco.onnx shoule be in the onnx_files folder. And then `python detect_with_onnx.py --weight=onnx_files/res50_coco.onnx`, then the config above should show as `res50_coco`, but in your screen snap, the config is res101_coco. There must be something wrong.

**Comment by Sandeep418** [comment]
I tried with resnet-50 and resnet-101 both predictions were not good. please can you share your onnx weight file.

**Comment by feiyuhuahuo** [comment]
https://drive.google.com/drive/folders/1bMIlDLGJPAlBRaSpJbS7lTGvEHBsaDro?usp=sharing
Besides, my PyTorch version is 1.7.0

**Comment by Sandeep418** [comment]
Thanks for sharing.
I found the issue, I am getting extra opp constant when I was using torch version 1.6, then I converted with torch version 1.7.0 it was removed from your weight file. But I am not able to remove from my weight file still I am trying to remove the constant opp from my weight file.
![image (8)](https://user-images.githubusercontent.com/40691260/114373356-20112800-9ba0-11eb-84ad-dc5c908468e7.png)
